Source/Core/DolphinQt/GCMemcardManager.h

```diff
@@ -13,21 +13,26 @@
 
 #include "Common/CommonTypes.h"
 
+#include "Core/HW/GCMemcard/GCMemcardUtils.h"
```

Comment:
This can likely be replaced with forward declarations (the standard allows forward declaring types in std::vector).

Reply:
This is for `void ExportFiles(Memcard::SavefileFormat format);` which I don't think you can forward declare.

Reply:
It's permissible in the standard to forward declare enumerations that are either `enum class Name`, `enum class Name : <explicit type>`, or `enum Name : <explicit type>`

Reply:
Sure enough, that works. Good to know.